website/docs/r/eventgrid_topic.html.markdown

```diff
@@ -56,6 +58,14 @@ The following arguments are supported:
 
 ---
 
+A `identity` block exports the following:
```

Comment:
```suggestion
A `identity` block supports the following:
```

We should also call out `identity_ids`

And in the export section of the docs, we want to call out `principal_id` and `tenant_id`